CONTRIBUTORS.md

```diff
@@ -25,8 +25,9 @@ doing a little due-diligence:
 - See if your use case is covered in the manual.
 - See if your use case is demonstrated by the API docs.
 - Ask the mailing list about your particular use case.
-- Discuss your issue with someone in one of the IRC channels: `#zftalk` (general help) or
-  `#zftalk.dev` (contributors) on [Freenode IRC](http://freenode.net).
+- Discuss your issue with someone in one of the public channels: 
+  [Slack](https://zendframework-slack.herokuapp.com/) or on the
```

Comment:
The same comment here:

> I prefer the word "Chat", because if we use "Slack" then we must also use "Discourse" for "Forum" etc.
**But who forces us to use company names, rather than the names which are intelligible to all?**

And if tomorrow Slack is outdated or and no longer a thing for _hipsters_, then we must change this again. 😉 

https://github.com/zendframework/maintainers/pull/40#discussion_r165924125